content/blog/wireshark-protobuf-and-grpc-dissectors.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Wireshark Protobuf and gRPC dissectors
+date: 2020-10-20
+spelling: cSpell:ignore Huang Qiangxiong
+author:
+  name: Huang Qiangxiong
+---
+
+[Wireshark](https://www.wireshark.org/) is the most famous open source network protocol analyzer, which is widely used in network troubleshooting, analysis, software and communications protocol development, and education. Here is the history of Wireshark Protobuf and gRPC dissectors:
+
+- Wireshark 2.6.0 - adds the initial version of Protobuf and gRPC dissectors.
+
+- Wireshark 3.2.0 - supports dissecting serialized Protobuf data (such as gRPC) more precisely according to *.proto files, and supports dissecting gRPC messages of streaming calls.
+
+- Wireshark 3.3.0 - fixes some bugs about parsing *.proto files and supports some new features about Protobuf dissector.
```

Comment:
An excerpt of a changelog isn't really appropriate. What's your _intent_ behind the inclusion of these log entries? What is the essence of the information that you are trying to conveny?

Reply:
I think the content here is actually quite helpful, as Arch has Wireshark 3.3, Ubuntu 20.04LTS has Wireshark 3.2, Ubuntu 18.04LTS has 2.4, Debian stable is on 2.6. Users following this will get quite a different experience depending on their version.

Now there is probably a better way to preface this, so it flows and is more clear what the purpose is and the key take aways.

Reply:
I have moved the history to a section near the end. How about that?